android-plugin/src/main/resources/org/sonar/l10n/java/rules/squid/SPRI002_java.html

```diff
@@ -0,0 +1,13 @@
+<img src="http://www.neomades.com/extern/partage/ecoCode/2sur5_1x.png">
+<p>
+    Importing the <code>com.google.android.gms.analytics.Tracker</code> class means that the app sends hits to Google Analytics. It is not necessarily sensitive information, but it is a first step towards Google Ads and hence this practice should be discouraged at early stage.
```

Comment:
Why do we not talk about "import com.google.firebase.analytics" in this paragraph?